packages/pagination/src/PaginationContent.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Util } from 'reactstrap';
+import BlockUI from 'react-block-ui';
+import { usePagination } from './Pagination';
+
+const PaginationContent = ({
+  component: Component,
+  loadingMessage,
+  itemKey,
+  loader = false,
+  ...rest
+}) => {
+  const { page, loading } = usePagination();
+
+  return (
+    <BlockUI
+      data-testid="pagination-content-con"
```

Comment:
is it ok to have `data-testid` here? Seems like testing spilling over.

Reply:
`data-testid` is just a simple prop for `react-testing-library` to find components in the DOM Tree rather than by querySelector or Text.  This way if any of the underlying markup or classes change this will always be the sole identifier of the component for testing.